tests/model_registry/test_rest_api.py

```diff
@@ -0,0 +1,20 @@
+import schemathesis
+import pytest
+from schemathesis.checks import ignored_auth
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+schema = schemathesis.from_pytest_fixture("generate_schema")
```

Comment:
why do we need to wrap with calling the fixture and not using `from_uri` with the uri?

Reply:
Because I want it to use the real host of the MR instance rather than grab the localhost from the openAPI spec. In order to do that and pass it as a base_url, I need it to be in a fixture so that I can wait for the MR instance and grab it.
If I were to call it as from_uri here, it would not be doable, as this happens before the tests are run and it does not have the ability to see the fixtures yet.